tests/files/packet_centos7-docker-weave-upgrade-ha.yml

```diff
@@ -7,10 +7,5 @@ mode: ha
 kube_network_plugin: weave
 kubernetes_audit: true
 
-# Docker specific settings:
-container_manager: docker
-etcd_deployment_type: docker
-resolvconf_mode: docker_dns
```

Comment:
The job name contains `docker`.
If using the default `containerd` instead of `docker`, it is necessary to rename the job to remove `docker`.

Reply:
argh, I focused too much on getting the debian job running, will fix this as well and do another CI run